lib/diagnostic.js

```diff
@@ -60,7 +60,7 @@ let deathStar = {
 // Question 4: Write the code that would set the `deathStar`'s `fire` method
 // in the context of `alderaan` but does not invoke the method,
 // so that we could call answer4() // Alderaan
-let answer4 = null;  // replace null with your response
+let answer4 = new deathStar();  // replace null with your response
```

Comment:
We want to bind `deathStar`'s `fire()` method to `alderaan`, not create a new instance of `deathStar`.

```javascript
deathStar.fire.bind(alderaan);
```